docs/index.md

```diff
@@ -132,6 +134,33 @@ Run the following command to report each incident:
     --timestampResolved "<timestampResolved>"
 ```
 
+### Events Helpers
```

Comment:
does it make sense to update the examples as well?

Reply:
Yes. Let's do it on a separate pull request, as I'd also like to move the examples to a dedicated page (this is because right now this page is already quite long and is becoming harder to navigate).